runtime/autoload/provider/pythonx.vim

```diff
@@ -13,19 +13,21 @@ function! provider#pythonx#Detect(ver) abort
   let skip = exists(skip_var) ? {skip_var} : 0
   if exists(host_var)
     " Disable auto detection
-    let [check, err] = s:check_interpreter({host_var}, a:ver, skip)
+    let [check, err, _] = s:check_interpreter({host_var}, a:ver, skip)
     return check ? [{host_var}, err] : ['', err]
   endif
 
   let detect_versions = (a:ver == 2) ?
-        \   ['2.7', '2.6', '2', '']
-        \ : ['3.5', '3.4', '3.3', '3.2', '3', '']
+        \   ['2', '2.7', '2.6', '']
```

Comment:
@fwalch Yes, in my early issue (#1334) up to now it wouldn't have worked but now I have adopted a "saner" setup. And as long as fully specified version of python interpreter (e.g. python2.7) are checked it will work